app/controllers/public/sessions/index.js

```diff
@@ -0,0 +1,20 @@
+import Ember from 'ember';
+import { chain } from 'lodash';
+
+const { Controller, computed } = Ember;
+
+export default Controller.extend({
+  sessionsByTracks: computed('model.sessions.[]', 'selectedTrackId', function() {
```

Comment:
This entire computed property to be removed and dynamic segments to be used instead of it to load sessions based on tracks.